### PR TITLE
[ERSSUP-61463]-[]-[Add all env from e-RS API, amend azure pipelines a…

### PR DESCRIPTION
**Pull request by nhsd-radoslav-radev**
…nd proxy target endpoints]-[RR]

## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: proxies/live/apiproxy/policies/ServiceCallout.CallHealthcheckEndpoint.xml [hunk 1]
```diff
@@ -4,24 +4,19 @@
...
-            <Property name="request.retain.headers">User-Agent,Referer,Accept-Language</Property>
-            <Property name="retain.queryparams">apikey</Property>
-        </Properties>
+        <SSLInfo>
+            <Enabled>true</Enabled>
+        </SSLInfo>
+        <LoadBalancer>
+            <Server name="{{ ERS_TARGET_SERVER | default('e-referrals-service-patient-care-api') }}"/>
+        </LoadBalancer>
+        <Path>/ers-api/_ping</Path>
```
Comment (nhsd-radoslav-radev) [inline proxies/live/apiproxy/policies/ServiceCallout.CallHealthcheckEndpoint.xml:19]: Replace with /v1/_ping

File: proxies/live/apiproxy/targets/target.xml [hunk 2]
```diff
@@ -19,25 +23,26 @@
...
-    </HTTPTargetConnection>
-  -->
-  <HTTPTargetConnection>
-    <URL>http://mocktarget.apigee.net</URL>
-    <Properties>
-      <Property name="supports.http10">true</Property>
-      <Property name="request.retain.headers">User-Agent,Referer,Accept-Language</Property>
-      <Property name="retain.queryparams">apikey</Property>
-    </Properties>
+      <Path>/v1</Path>
```
Comment (nhsd-radoslav-radev) [inline proxies/live/apiproxy/targets/target.xml:41]: Speak with Partha about where the API will sit

Comment (nhsd-radoslav-radev) [reply proxies/live/apiproxy/targets/target.xml:41]: Fhir r4 should be the path

**Review by nhsd-david-wass — APPROVED** [review]

**Review by nhsd-david-wass — APPROVED** [review]

**Pull request merged**